plugins/community/nipunn1313-mypy-grpc/v3.5.0/requirements.txt

```diff
@@ -0,0 +1,3 @@
+mypy-protobuf==3.5.0
+protobuf==4.23.4
+types-protobuf==4.23.0.2
```

Comment:
These updates are not yet automated. I did the following to update this file:

1. Removed `&& pip uninstall --yes pip setuptools \` line from Dockerfile.
2. Ran `docker build --target build -t test .`
3. Ran `docker run -it --rm test /bin/sh`
4. In the container, ran `. bin/activate` to activate the virtualenv, ran `pip list --outdated` and `pip upgrade ...` to upgrade things, and ran `pip freeze` to generate a new requirements.txt file.